api/base/serializers.py

```diff
@@ -803,6 +803,25 @@ def is_valid(self, clean_html=True, **kwargs):
 
         return ret
 
+class JSONAPIRelationshipsSerializer(ser.Serializer):
+    """Base Relationship serializer. Requires that a `type_` option is set on `class Meta`.
+    Provides a simplified serialization of the relationship, allowing for simple update request
+    bodies.
+    """
+
+    def to_representation(self, obj):
+        meta = getattr(self, 'Meta', None)
+        type_ = getattr(meta, 'type_', None)
+        assert type_ is not None, 'Must define Meta.type_'
+
+        relation_id_field = self.fields['id']
```

Comment:
This assumes that the serializer has an `id` field. Would it make sense to put the `id` and `type` fields on this class and get rid of `RelationshipResourceIdObjectSerializer`?